arelle/plugin/validate/ESEF/DTS.py

```diff
@@ -26,7 +26,7 @@
                     linkbaseRefTypes, filenamePatterns, filenameRegexes)
 from .Util import isExtension
 from arelle.ValidateXbrl import ValidateXbrl
-from .common import TypeGetText
```

Comment:
We're going to [need this](https://github.com/Arelle/Arelle/blob/da6cce5e40246e6db393e0c5363d718abcd25657/arelle/Cntlr.py#L321) in the `./<repo-root>/arelle` directory. May as well move it up now.